build.sbt

```diff
@@ -34,4 +23,40 @@ resolvers += "scalaz-bintray" at "http://dl.bintray.com/scalaz/releases"
 
 javaOptions in Test += "-Dconfig.file=conf/application.test.conf"
 
-playDefaultPort := 9101
+PlayKeys.playDefaultPort := 80
+
+// Allow building both ways on Teamcity
+addCommandAlias("riffRaffArtifact", "riffRaffUpload")
+
+packageName in Universal := name.value
+maintainer := "Guardian Developers <dig.dev.software@theguardian.com>"
+packageSummary := "AMIable"
+packageDescription := """Web app for monitoring the use of AMIs"""
+debianPackageDependencies := Seq("openjdk-8-jre-headless")
+
+javaOptions in Universal ++= Seq(
+  "-Dpidfile.path=/dev/null",
+  s"-Dconfig.file=/etc/${name.value}.conf",
+  "-J-XX:MaxRAMFraction=2",
+  "-J-XX:InitialRAMFraction=2",
```

Comment:
Yeah, it won't matter here. Be good to know how to set it to arbitrary fractions for stuff that will care more. 512 -> 768 is a big difference on a micro and 256 -> 384 might be the difference between working and not working on a nano. (I assume it's fairly easy)